Data/ActionHandler/CreateMarriageHandler.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace CommonGateway\HuwelijksplannerBundle\ActionHandler;
+
+use CommonGateway\CoreBundle\ActionHandler\ActionHandlerInterface;
+use CommonGateway\HuwelijksplannerBundle\Service\CreateMarriageService;
+use Symfony\Component\Security\Core\Security;
+
+class CreateMarriageHandler implements ActionHandlerInterface
+{
+    private CreateMarriageService $createMarriageService;
+    private Security $security;
+
+    public function __construct(CreateMarriageService $createMarriageService, Security $security)
+    {
+        $this->createMarriageService = $createMarriageService;
+        $this->security = $security;
+    }
+
+    /**
+     *  This function returns the requered configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @throws array a [json-schema](https://json-schema.org/) that this  action schould comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'        => 'https://vng.opencatalogi.nl/schemas/hp.huwelijk.schema.json',
+            '$schema'    => 'https://json-schema.org/draft/2020-12/schema',
+            'title'      => 'CreateMarriage',
+            'required'   => ['huwelijksEntityId', 'assentEntityId', 'klantEntityId', 'natuurlijkPersoonEntityId'],
+            'properties' => [
+                'huwelijksEntityId' => [
```

Comment:
We can use hardcoded functions that use this ref to find these entities instead of adding them to the Action configuration?

![image](https://user-images.githubusercontent.com/3785488/216957665-6e569f7c-6eac-4490-9b1e-df8b21e523be.png)

Reply:
Add a // todo ?